assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import classNames from 'classnames';
+import { parseISO } from 'date-fns';
+import { __ } from '@wordpress/i18n';
+
+import { DisplayStartOrEndDate } from '@sharedServices/filterState';
+import { PLUS_ONE_MONTH, PLUS_TWO_MONTHS } from '@sharedConstants/defaultDates';
+import { BiggieCalendarDate, CalendarDateRange } from '../';
+
+interface Props {
+	className?: string;
+	displayDate: DisplayStartOrEndDate;
+	endDate: string;
+	footerText?: string;
+	headerPrefix?: string;
```

Comment:
let's remove this prop because we'll handle the text differently at the source

Reply:
it removed, not sure why it's showing

Reply:
seems to be outdated

![image](https://user-images.githubusercontent.com/1477453/77671078-c091ab80-6f8f-11ea-8aee-8a1ebbc431b7.png)